api/v1alpha1/openapi.yaml

```diff
@@ -3432,11 +3432,82 @@ components:
         - metadata
         - items
       description: FleetList is a list of Fleets.
+
+    DisruptionAllowance:
+      type: object
+      properties:
+        groupBy:
+          type: array
+          items:
+            type: string
+          description: List of labels to perform grouping for the disruption allowance.
+        minAvailable:
+          type: integer
+          description: The maximum number of unavailable devices allowed during rollout.
+        maxUnavailable:
+          type: integer
+          description: The minimum number of required available devices during rollout.
+      description: DisruptionAllowance defines the level of allowed disruption when rollout is in progress.
+
+    Percentage:
+      type: string
+      format: '^(?:(?:[1-9][0-9]?)|100|0)%$'
+      description: Percentage is the string format representing percentage string.
+
+    Batch:
+      type: object
+      properties:
+        selector:
+          $ref: '#/components/schemas/LabelSelector'
+        successThreshold:
+          $ref: '#/components/schemas/Percentage'
+        limit:
+          oneOf:
+            - $ref: '#/components/schemas/Percentage'
+            - type: integer
+              minimum: 1
+      description: Batch is an element in batch sequence.
+    Duration:
+      type: string
+      pattern: '^[1-9]\d*[smhd]$'
+      description: |
+        The maximum duration allowed for the action to complete. 
+        The duration should be specified as a positive integer 
+        followed by a time unit. Supported time units are:
+        - 's' for seconds
+        - 'm' for minutes
+        - 'h' for hours
+        - 'd' for days
+
+    BatchSequence:
+      type: array
+      minItems: 1
+      items:
+        $ref: '#/components/schemas/Batch'
+      description: BatchSequence defines the list of batches to be executed in sequence.
+
+    RolloutPolicy:
+      type: object
+      properties:
+        disruptionAllowance:
+          $ref: '#/components/schemas/DisruptionAllowance'
+        batchSequence:
```

Comment:
We said that we'll have a property to allow different strategies in the future, other than batchSequence.

Reply:
Currently it is kind of 'dangling' property since the only strategy is batch.  I think we should have it once we have additional strategy.

Reply:
Avishay's right, We cannot have `batchSequence` at this level, it belongs below the selection strategy. How about
```
rolloutPolicy:
  selectionStrategy:
    type: batchSequence
    sequence:
    - ...
```

Reply:
We agreed on device selection strategy on the top level of rollout policy.
I added it here: https://github.com/flightctl/flightctl/compare/e74eda546f0ed592cb7f1e0364a7d32c9e3afca5..2c4662d3ad83abae5a0df1bbbf8e64e8d1fc53cd

Reply:
Although I can change to what @fzdarsky suggested